frontend/src/v5/store/tickets/card/ticketsCard.selectors.ts

```diff
@@ -20,11 +20,11 @@ import { SequencingProperties, TicketBaseKeys, TicketsCardViews } from '@/v5/ui/
 import { createSelector } from 'reselect';
 import { selectTemplateById, selectTemplates, selectTicketById, selectTickets } from '../tickets.selectors';
 import { ITicketsCardState } from './ticketsCard.redux';
-import { getTicketIsCompleted } from './ticketsCard.helpers';
 import { DEFAULT_PIN, getPinColorHex, formatPin } from '@/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.helpers';
```

Comment:
I don't think the chip should be truncated here 
![image](https://github.com/3drepo/3drepo.io/assets/43547703/1ec5ca4c-2401-4bfe-a0a3-af58e19852e3)